superset/assets/javascripts/SqlLab/components/SqlEditor.jsx

```diff
@@ -83,7 +83,7 @@ class SqlEditor extends React.PureComponent {
     this.props.actions.setActiveSouthPaneTab('Results');
   }
   stopQuery() {
-    this.props.actions.stopQuery(this.props.latestQuery);
```

Comment:
is the `stopQuery` action used elsewhere? if not we should remove it in `superset/assets/javascripts/SqlLab/actions.js` as well